resources/todomvc/vanilla-examples/javascript-es5/src/view.js

```diff
@@ -89,17 +85,22 @@
         });
     };
 
+    View.prototype._replaceContentWithHtml = function (element, html) {
+        const parsedDocument = new DOMParser().parseFromString(html, "text/html");
+        element.replaceChildren(...parsedDocument.body.childNodes);
```

Comment:
This is the easiest I found to add the resiult of DOMParser to the DOM. Actual libraries usually use more code because they want to insert among other sieblings, but we don't need this here.